.bazelrc

```diff
@@ -22,14 +22,6 @@ build:generic_clang --repo_env=CC=clang
 build:generic_clang --cxxopt=-std=c++17
```

Comment:
Above, in L4, consider teh GitHub warning for spelling of the word "unfortunat**t**ely"

Reply:
Fixed.